packages/lime-system/tests/test_lime_network.lua

```diff
@@ -76,6 +76,35 @@ describe('LiMe Network tests', function()
         network.get_mac:revert()
     end)
 
+    it('test network.configure() with only lime.proto.lan', function()
+        local ifname = 'eth99'
+        config.set('system', 'lime')
+        config.set('system', 'domain', 'lan')
+        config.set('network', 'lime')
+        config.set('network', 'primary_interface', ifname)
+        config.set('network', 'main_ipv4_address', '10.%N1.0.0/16')
+        config.set('network', 'main_ipv6_address', '2a00:1508:0a%N1:%N200::/64')
+        config.set('network', 'protocols', {'lan'})
+        config.set('network', 'resolvers', {'4.2.2.2'})
+        config.set('wifi', 'lime')
+        config.set('wifi', 'ap_ssid', 'LibreMesh.org')
+        uci:commit('lime')
+
+        stub(network, "get_mac", function () return  {'00', '00', '00', '00', '00', '00'} end)
+        stub(network, "scandevices", function () return  {eth99={}} end)
+        stub(utils, "getBoardAsTable", function () return BOARD end)
+
+        test_utils.disable_asserts()
+        network.configure()
+        test_utils.enable_asserts()
+
+        assert.is.equal("1500", uci:get("network", "lan", "mtu"))
```

Comment:
Is this a default value? it is not self-documented, at least for me...

Reply:
It is hardcoded in `lime.proto.lan`: `22:	uci:set("network", "lan", "mtu", "1500")`.

My idea behind "testing this" was exactly to make it clear that we have a fixed value there, without judging if it is good or not. And to serve as example on how to test this things, not because I believe in `1500` ;)